### PR TITLE
Add admin dropdown in navbar

### PR DESCRIPTION
**Pull request by xzhflying**
* Modify default layouts theme

### CODE REVIEW

File: app/themes/default/views/layouts/default.html.slim [hunk 1]
```diff
@@ -38,6 +38,13 @@ html
             li
               = link_to(t('layout.navbar.help'), '#')
             - if user_signed_in?
+              - if current_user.administrator?
+                li
+                  a.dropdown-toggle data-toggle="dropdown" = current_user.name
+                    b.caret
+                  ul.dropdown-menu
+                    li
+                      = link_to(t('layout.navbar.admin_panel'), admin_path)
```
Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: Does this apply to everyone?

File: config/locales/en/layout.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ en:
       register: 'Register'
       sign_in: 'Sign In'
       sign_out: 'Sign Out'
+      admin_panel: 'Administrator Panel'
```
Comment (lowjoel) [inline config/locales/en/layout.yml]: Administration

Comment (lowjoel) [inline config/locales/en/layout.yml]: Neither 'admin' nor 'administrator'

File: app/themes/default/views/layouts/default.html.slim (hunk 1 above)
Comment (xzhflying) [inline app/themes/default/views/layouts/default.html.slim]: I'm not sure if I can use `.administrator?` to judge if a user is administrator. Maybe I should use `can? :mange, :all`?

File: config/locales/en/layout.yml (hunk 2 above)
Comment (xzhflying) [inline config/locales/en/layout.yml]: got it

File: app/themes/default/views/layouts/default.html.slim (hunk 1 above)
Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: What is the permission needed to determine if the user is an administrator?

Comment (xzhflying) [inline app/themes/default/views/layouts/default.html.slim]: user's role?

Comment (allenwq) [inline app/themes/default/views/layouts/default.html.slim]: I think it should be `can? :manage current_tenant`

Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: And how about a global admin panel?

**Comment by lowjoel** [comment]
Please rebase.

File: app/themes/default/views/layouts/default.html.slim (hunk 1 above)
Comment (allenwq) [inline app/themes/default/views/layouts/default.html.slim]: Currently we haven't split instance and global admin yet.

Once we have the global admin panel it should be `user.administrator?` or `can?(:manage, :all)`

Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: I'd be incredibly careful with the `:manage` permission.

Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: I think we should use `can?(:manage, :all)` That's testing effect and not implementation.

Comment (xzhflying) [inline app/themes/default/views/layouts/default.html.slim]: So... change it into `can?(:manage, :all)` for now?

Comment (allenwq) [inline app/themes/default/views/layouts/default.html.slim]: @lowjoel does `manage :all` works for instance admin ?

Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: So there's some dissonance here. This is the global admin but actually it should check the instance permissions... interesting.

Comment (allenwq) [inline app/themes/default/views/layouts/default.html.slim]: Yah, because this is shared

Comment (xzhflying) [inline app/themes/default/views/layouts/default.html.slim]: Means the same button leads to different pages for different admins?

Comment (allenwq) [inline app/themes/default/views/layouts/default.html.slim]: @xzhflying #250

Comment (allenwq) [inline app/themes/default/views/layouts/default.html.slim]: @lowjoel  So `can?(:manage, current_tenant)` ? this works for both global admin and instance admin.

Comment (lowjoel) [inline app/themes/default/views/layouts/default.html.slim]: Okay.

Comment (xzhflying) [inline app/themes/default/views/layouts/default.html.slim]: This has already been changed into `can?(:manage, current_tenant)`

**Comment by lowjoel** [comment]
Thanks!

**Pull request merged**